### PR TITLE
Added Factorial calculator files

### PR DESCRIPTION
**Pull request by Anchita15**
### 👨‍💻 Changes proposed

Added Factorial calculator files.
Its responsive.
This calculator is created using HTML, CSS, JavaScript and gives an good insight in to these tech stacks.

### ✅ Check List (Check all the applicable boxes) 

- [✅] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [✅] This PR does not contain plagiarized content.
- [✅] My Addition/Changes works properly and matches the overall repo pattern.
- [✅] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots

Factorial SS

### CODE REVIEW

**Comment by NOiR-07** [comment]
add your project in the index file of website too

**Comment by Anchita15** [comment]
> add your project in the index file of website too

Okay, I'll do.

**Comment by Anchita15** [comment]
> add your project in the index file of website too

I have done the required changes as instructed. You can check through.

File: assets/Images/factorial_calculator/index.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
+body {
```
Comment (Astrodevil) [inline assets/Images/factorial_calculator/index.css:1]: Put this Css file in proper directory

**Comment by NOiR-07** [comment]
update your forked repo to resolve conflict

**Comment by NOiR-07** [comment]
Make a new pr, closing this now

**Pull request closed without merging**